common/static/common/js/discussion/utils.js

```diff
@@ -184,10 +184,8 @@
             if (!params.error) {
                 params.error = function() {
                     self.discussionAlert(
-                        gettext('Sorry'),
-                        gettext(
-                            'We had some trouble processing your request. Please ensure you have copied any ' +
-                            'unsaved work and then reload the page.')
+                        gettext('Error'),
```

Comment:
Ah, I see. If so, we need to put the Error back. sorry, false alarm. I though it was one sentence. Here is how it looks.

<img width="606" alt="screen shot 2016-12-05 at 12 20 04 pm" src="https://cloud.githubusercontent.com/assets/16127998/20894937/61118e80-bae5-11e6-808e-d7121f0ca29d.png">